src/api/inline-api.ts

```diff
@@ -178,23 +178,25 @@ export class DataviewInlineApi {
      * Takes a filename and arbitrary input data.
      */
     public view(viewName: string, input: any) {
-        /** This cannot be used on systems without file access (i.e. web and mobile devices). */
-        if (!(this.app.vault.adapter instanceof FileSystemAdapter)) {
-            renderErrorPre( this.container, `Dataview: file system access is not available.` );
+
+        let currentFile = this.app.workspace.getActiveFile();
```

Comment:
This may not be the file the inline block is executing - use `this.currentFilePath`, which is guaranteed to be non-null.